mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -91,80 +61,89 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
     return data_thresholded, center_locations
 
 
-def gridding_2d(points, values, img_shape, method='linear', point_min=None,
-                point_max=None):
+def grided_inverse_fourier_transform_nd(points, values, grid, method):
     """
-    Interpolate non-Cartesian data into a cartesian grid
+    This function calculates the grided Inverse fourier transform
```

Comment:
same typo here gridded.

Reply:
I am still confused on the spelling of gridded, will update this